src/java.base/share/classes/java/text/NumberFormat.java

```diff
@@ -183,14 +171,16 @@
  * If multiple threads access a format concurrently, it must be synchronized
  * externally.
  *
- * @implSpec The {@link #format(double, StringBuffer, FieldPosition)},
+ * @implSpec
+ * <h3>Null Parameter Handling</h3>
```

Comment:
Not sure we'd want this as a h3 header, as it makes `@implSpec` less significant.